pkgs/development/libraries/ogre/1.10.x.nix

```diff
@@ -6,7 +6,20 @@
 , libXxf86vm, libICE
 , libXrender
 , withNvidiaCg ? false, nvidia_cg_toolkit
-, withSamples ? false }:
+, withSamples ? false
+, writeScriptBin
+, rsync
+, libobjc
```

Comment:
This shouldn't be needed with Foundation.

Reply:
This is needed because the build fails when it could not find `libobjc`.